tests/util/ginkgo/lib.go

```diff
@@ -514,12 +514,29 @@ func (ns NsWrapper) CqlExecute(podName string, stepDesc string, cql string, user
 	ns.ExecVPanic(k)
 }
 
-func (ns *NsWrapper) CheckForCompletedCassandraTasks(dcName, command string, count int) {
+func (ns *NsWrapper) CheckForCassandraTasks(dcName, command string, completed bool, count int) {
 	step := fmt.Sprintf("checking that cassandratask command %s has succeeded", command)
 	json := "jsonpath={.items[*].spec.jobs[0].command}"
 	k := kubectl.Get("cassandratask").
-		WithLabel(fmt.Sprintf("cassandra.datastax.com/datacenter=%s", dcName)).
-		WithLabel("control.k8ssandra.io/status=completed").
-		FormatOutput(json)
+		WithLabel(fmt.Sprintf("cassandra.datastax.com/datacenter=%s", dcName))
+
+	if completed {
```

Comment:
Nit: when `completed` is false, I would have expected the `control.k8ssandra.io/status` to NOT be equal to `completed`. Here instead, we retrieve all tasks, both complete and incomplete.

Reply:
That was the intention. The idea was to fetch all possible tasks to ensure no extra ones are created.

Reply:
Maybe rename to `onlyComplete` for clarity?